packages/cactus-plugin-ledger-connector-iroha/src/test/typescript/integration/run-transaction-endpoint-v1.test.ts

```diff
@@ -0,0 +1,786 @@
+    t.ok(res);
+    t.ok(res.data);
+    t.equal(res.status, 200);
+    console.log(res.data.transactionReceipt);
+    t.equal(res.data.transactionReceipt.status, "COMMITTED");
+  }
+
+  // //  Use Promise.race to cancel the promise
+  // {
+  //   const req1 = {
+  //     commandName: "producePendingTx",
+  //     params: [],
+  //   };
+  //   Promise.race([
+  //     //FIXME - the Iroha Javascript SDK does not give any output if we try to produce a pending transaction
+  //     // This results in an infinite loop and thus the following code cannot be executed.
+  //     // This fix is not perfect. It cancels the request with a timeout, but will result in grpc "Error: 14 UNAVAILABLE: GOAWAY received
+  //     // Once the Iroha Javascript SDK is justitied. We can safely produce a pending transaction.
+  //     apiClient.runTransactionV1(req1 as RunTransactionRequest),
+  //     new Promise((resolve) => setTimeout(resolve, 1000)),
+  //   ]);
+  // }
+
+  // {
+  //   const req = {
+  //     commandName: "getPendingTransactions",
+  //     params: [5, undefined],
+  //   };
+  //   const res = await apiClient.runTransactionV1(req as RunTransactionRequest);
+  //   t.ok(res);
+  //   t.ok(res.data);
+  //   t.equal(res.status, 200);
+  //   t.deepEqual(
+  //     res.data.transactionReceipt[0].payload.reducedPayload.commandsList[0],
+  //     { createAsset: { assetName: "coinb", domainId: "test", precision: 3 } },
+  //   );
+  //   t.equal(
+  //     res.data.transactionReceipt[0].signaturesList[0].publicKey,
+  //     "313a07e6384776ed95447710d15e59148473ccfc052a681317a72a69f2a49910",
+  //   );
+  // }
+
+  // use bluebird to cancel Promise
+  // {
+  //   const req1 = {
+  //     commandName: "producePendingTx",
+  //     params: [],
+  //   };
+  //   const promise = apiClient.runTransactionV1(req1 as RunTransactionRequest);
+  //   const p2 = new Promise((onCancel) => {
+  //     promise;
+  //     onCancel(() => console.log("p2 canceled"));
+  //   });
+  //   p2.cancel();
+  // }
+
+  // // {
+  // //   const req = {
+  // //     commandName: "removePeer",
+  // //     params: [
+  // //       "0000000000000000000000000000000000000000000000000000000000000002",
+  // //     ],
+  // //   };
+  // //   const res = await apiClient.runTransactionV1(req as RunTransactionRequest);
+  // //   console.log(res.data.transactionReceipt);
+  // // }
+
+  // // {
+  // //   const req = {
+  // //     commandName: "fetchCommits",
+  // //     params: [],
+  // //   };
+  // //   const res = await apiClient.runTransactionV1(req as RunTransactionRequest);
+  // //   t.ok(res);
+  // //   t.ok(res.data);
+  // //   t.equal(res.status, 200);
+  // //   console.log(res.data.transactionReceipt);
+  // // }
+
+  t.end();
+});
+
+test("AFTER " + testCase, async (t: Test) => {
+  const pruning = pruneDockerAllIfGithubAction({ logLevel });
+  await t.doesNotReject(pruning, "Pruning didn't throw OK");
+  t.end();
+});
```

Comment:
at the file I see hardcoded names of users, assets etc. used many times - I suggest to use constants

Reply:
Thank you for pointing out! Now I used const for users.

Reply:
Also put my best effort to remove every magical strings including using const for assets also. :D